aggregator/handler/config.go

```diff
@@ -135,14 +153,62 @@ func (c *writerConfiguration) NewWriterOptions(
 }
 
 type flushHandlerConfiguration struct {
-	// Flushing handler type.
-	Type Type `yaml:"type"`
+	// StaticBackend configures the backend.
+	StaticBackend *staticBackendConfiguration `yaml:"staticBackend"`
+
+	// DynamicBackend configures the dynamic backend.
+	DynamicBackend *dynamicBackendConfiguration `yaml:"dynamicBackend"`
+}
 
-	// Backend configures the backend.
-	Backend *backendConfiguration `yaml:"backend"`
+func (c flushHandlerConfiguration) Validate() error {
+	if c.StaticBackend == nil && c.DynamicBackend == nil {
+		return errors.New("neither dynamic nor static backend was configured")
+	}
+	if c.StaticBackend == nil && c.DynamicBackend == nil {
```

Comment:
This seems like it should be `if c.StaticBackend != nil && c.DynamicBackend != nil {`, can we add a test?

Reply:
good catch, will do